README.md

```diff
@@ -86,34 +125,88 @@ The following grammar represents the Stage 1 criteria, which is:
 5. No separator adjacent to `0b`, `0B`, `0o`, `0O`, `0x`, `0X`.
 
 
+
+Changes to [11.8.3 Numeric Literals](https://tc39.github.io/ecma262/#prod-NumericLiteral)
+
 ```
-NumericLiteralSeparator ::
-  _
-
-DecimalDigits ::
-  DecimalDigit
-  DecimalDigit NumericLiteralSeparator DecimalDigit
-  DecimalDigits DecimalDigit
-
-BinaryDigits ::
-  BinaryDigit
-  BinaryDigit NumericLiteralSeparator BinaryDigit
-  BinaryDigits BinaryDigit  
-
-OctalDigits ::
-  OctalDigit
-  OctalDigit NumericLiteralSeparator OctalDigit
-  OctalDigits OctalDigit    
-
-HexDigits ::
-  HexDigit
-  HexDigit NumericLiteralSeparator HexDigit
-  HexDigits HexDigit  
++ NumericLiteralSeparator ::
++   _
+
+  DecimalDigits ::
+    DecimalDigit
++   DecimalDigit NumericLiteralSeparator DecimalDigit
+    DecimalDigits DecimalDigit
+
+  BinaryDigits ::
+    BinaryDigit
++   BinaryDigit NumericLiteralSeparator BinaryDigit
+    BinaryDigits BinaryDigit  
+
+  OctalDigits ::
+    OctalDigit
++   OctalDigit NumericLiteralSeparator OctalDigit
+    OctalDigits OctalDigit    
+
+  HexDigits ::
+    HexDigit
++   HexDigit NumericLiteralSeparator HexDigit
+    HexDigits HexDigit  
 ```
 
-### Standard library
+Changes to [7.1.3.1 ToNumber Applied to the String Type](https://tc39.github.io/ecma262/#sec-tonumber-applied-to-the-string-type) & [7.1.3.1.1Runtime Semantics: MV](https://tc39.github.io/ecma262/#sec-runtime-semantics-mv-s)
+
+```diff
+  StrDecimalLiteral:::
+    StrUnsignedDecimalLiteral
+    + StrUnsignedDecimalLiteral
+    - StrUnsignedDecimalLiteral
+
+  StrUnsignedDecimalLiteral:::
+    Infinity
+-   DecimalDigits . DecimalDigits_opt ExponentPart_opt 
+-   . DecimalDigits ExponentPart_opt 
+-   DecimalDigits ExponentPart_opt 
++   StrDecimalDigits . StrDecimalDigits_opt ExponentPart_opt 
++   . StrDecimalDigits ExponentPart_opt 
++   StrDecimalDigits ExponentPart_opt 
+
++ StrDecimalDigits ::
++   DecimalDigit
++   StrDecimalDigits DecimalDigit
+```
+
+Changes to [7.1.3.1.1Runtime Semantics: MV](https://tc39.github.io/ecma262/#sec-runtime-semantics-mv-s) shown in rendered proposal. 
```

Comment:
I like that suggestion: keeping all of the specification code in the spec doc and keeping the README file super short without the details of the implementaiton.

Want to do that in this PR or separately?